ecllibrary/std/File.ecl

```diff
@@ -105,6 +105,16 @@ EXPORT FsLandingZoneRecord := lib_fileservices.FsLandingZoneRecord;
 
 /*------------------------------------- Spray functions -----------------------------------------------------------*/
 
+/**
+ * Returns the boolean value of 'noCommon' property of the first DFUServer instance if it is defined,
```

Comment:
minor: comment is out of date (settings now in Global)

Reply:
Changed

Reply:
@AttilaVamos you marked this as resolved (you shouldn't be doing that as Jake has mentioned), and you haven't changed the comment.